CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [3.16.1] - 2021-02-23
```

Comment:
You did not update the compare tags list at the end of the file.

Reply:
Thanks :pray: